### PR TITLE
Feature/progress stepper

### PR DESCRIPTION
**Pull request by ONeill45**
- Added vertical stepper in place of previous dimension page navigation per the new design
- Added progress bar to track progress of both individual dimensions and overall assessment
- Added ability to navigate directly to a dimension by clicking
- Moved the active step in response to changing back via 'next' and 'prev'
- Added `container` class to survey to prevent survey from taking entire page

### CODE REVIEW

File: src/Components/AssessmentStepper/AssessmentStepper.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+export const AssessmentStepper = ({ dimArray, onStepClick, pages, model }) => {
+  const [activeStep, setActiveStep] = useState(0);
+  const [overallProgress, setOverallProgress] = useState(0);
+
+  useEffect(() => {
+    dimArray.map((dimension, index) => {
+      if (model.currentPageNo === getFirstPageOfDimension(dimension)) {
+        setActiveStep(index);
+      }
```
Comment (MitchellShimko) [inline src/Components/AssessmentStepper/AssessmentStepper.js]: Array.find() would be better here

**Review by MitchellShimko — APPROVED** [review]
looks great besides one suggestion

**Comment by Philip-Loeffler** [comment]
Screen Shot 2021-11-12 at 9 48 47 AM

would make this a little wider, so explainability and interpretability doesn't look like its touching the edge as much

**Comment by Philip-Loeffler** [comment]
Looks like in the design the progress bar is always visible so users can see how much they have completed for each section if they decide to skip around
Screen Shot 2021-11-12 at 9 55 32 AM

**Comment by Philip-Loeffler** [comment]
Mini progress bar for Project Details should probably increase if a user has filled out one of the text inputs as well. It is currently only going up if they have selected one of the radio buttons/checkboxes

**Comment by Philip-Loeffler** [comment]
Screen Shot 2021-11-12 at 10 07 44 AM

would update buttons on bottom to design. Really just need some border radius on them bad boys.

**Comment by Philip-Loeffler** [comment]
when clicking on a checkbox for question 69, under bias and fairness, all the checkboxes are selected
Screen Shot 2021-11-12 at 10 12 03 AM

this also occurs with question 83
Screen Shot 2021-11-12 at 10 14 27 AM

Have not seen this happening with any of the other checkboxes. just these two.

**Review by Philip-Loeffler — APPROVED** [review]

**Pull request merged**